server/ce2e/plugin_test.go

```diff
@@ -555,25 +551,233 @@ func TestSelectiveSync(t *testing.T) {
 				})
 				require.NoError(t, err)
 
+				err = mockClient.MockBatch("edit-message", []containere2e.BatchRequest{
+					{
+						Method: http.MethodPatch,
+						URL:    "/chats/" + newDMID + "/messages/" + newPostID,
+					}, {
+						Method: http.MethodGet,
+						URL:    "/chats/" + newDMID + "/messages/" + newPostID,
+					}},
+					[]containere2e.BatchResponse{
+						{
+							StatusCode: http.StatusNoContent,
+							Body: map[string]any{
+								"id":                   newPostID,
+								"messageType":          "message",
+								"createdDateTime":      time.Now().Format(time.RFC3339),
+								"lastModifiedDateTime": time.Now().Format(time.RFC3339),
+								"from": map[string]any{
+									"user": map[string]any{
+										"@odata.type":      "#microsoft.graph.teamworkUserIdentity",
+										"id":               "ms-" + tc.fromUser.Username,
+										"displayName":      tc.fromUser.Username,
+										"userIdentityType": "aadUser",
+										"tenantId":         "tenant-id",
+									},
+								},
+								"body": map[string]any{
+									"contentType": "text",
+									"content":     "Hello World",
+								},
+								"channelIdentity": map[string]any{
+									"channelId": newDMID,
+								},
+							},
```

Comment:
I see no difference between the three content created for the body, define as a variable and reduce code?